samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>This section is divided into two parts, preparing the training data and training the model.</p><p> </p><p>For <code>prepare_data()</code>, the argument 'batch_size' controls the number of training samples to be processed at one time. If you have a powerful GPU, this number can be further increased.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
corrected accordingly, Thanks.